### PR TITLE
Use SDK sample app instead

### PR DESCRIPTION
**Pull request by aornelas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # MoPub Unity Mediation
-  This repository contains the necessary Unity Packages to mediate ads through third-party networks. For a list of supported networks, visit the [MoPub Mediation Integration Tool](https://developers.mopub.com/docs/mediation/integrate/).
+This repository contains the necessary Unity Packages to mediate ads through third-party networks. For a list of supported networks, visit the [MoPub Mediation Integration Tool](https://developers.mopub.com/docs/mediation/integrate/).
+
+## Testing Mediation
+To test MoPub Unity Mediation, please start with these steps:
+1. Clone the [MoPub Unity SDK repository](https://github.com/mopub/mopub-unity-sdk)
+1. Open the Unity project under `unity-sample-app/` in the `mopup-unity-sdk` repository
+1. Import the desired packages from `unity-packages/` in this repository
```
Comment (rsatrasala) [inline README.md]: Import the desired packages from `unity-packages/` in [the current repository](https://github.com/mopub/mopub-unity-mediation)

Comment (aornelas) [reply README.md]: 👍

**Comment by rsatrasala** [comment]
We need the adapter versions to be bumped based on pull #58 and then use them here

**Comment by aornelas** [comment]
@rsatrasala reverted the package update since we'll do that separately (after [#58](https://github.com/mopub/mopub-ios-mediation/pull/58) merges)

this PR is ready for a final look!

**Pull request merged**